src/storage/SessionConfiguration.java

```diff
@@ -73,4 +79,42 @@ public String getLastLoginUsername() {
 	public void setLastLoginUsername(String lastLoginUsername) {
 		this.lastLoginUsername = lastLoginUsername;
 	}
+	
+	public void setLastLoginPassword(String lastPassword){
+		this.lastLoginPassword = encryptData(lastPassword);
+	}
+
+	public String getLastLoginPassword() {
+		// TODO Auto-generated method stub
+		return decryptData();
+	}
+
+	private String encryptData(String lastPassword) {
+		String result = "";
+		try {
+			String key = "HubTurboHubTurbo";
+		    Key aesKey = new SecretKeySpec(key.getBytes(), "AES");
+			Cipher cipher = Cipher.getInstance("AES");
+	        cipher.init(Cipher.ENCRYPT_MODE, aesKey);
+	        lastLoginEncrypted = cipher.doFinal(lastPassword.getBytes());
+	        result = new String(lastLoginEncrypted);
+	      } catch(Exception e) {
+	         e.printStackTrace();
+	      }
+		return result;
+	}
+
+	private String decryptData() {
+		String result = "";
+		try {
+			String key = "HubTurboHubTurbo";
+		    Key aesKey = new SecretKeySpec(key.getBytes(), "AES");
+	        Cipher cipher = Cipher.getInstance("AES");
+	        cipher.init(Cipher.DECRYPT_MODE, aesKey);
+	        result = new String(cipher.doFinal(lastLoginEncrypted));
+	      }catch(Exception e) {
+	         e.printStackTrace();
+	      }
```

Comment:
Catching `Exception` is bad because it lets any other exception type slip through. It may be a bit verbose in this case but I feel being more explicit is better.

Also, we should log the error instead of only printing the stack trace.